Functions/Administration/xASL_adm_CleanUpX.m

```diff
@@ -24,7 +24,9 @@
     % Zip all NIfTIs if we do not immediately run the pipeline afterwards
     if x.opts.bLoadData || ~x.opts.bProcessData
         if isfield(x.D,'ROOT')
-            xASL_adm_GzipAllFiles(x.D.ROOT,[],[],fullfile(x.opts.MyPath,'External'));
+			% Run zipping only on rawdata and derivatives and not on the entire folder
+            xASL_adm_GzipAllFiles(x.dir.Derivatives,[],[],fullfile(x.opts.MyPath,'External'));
+			xASL_adm_GzipAllFiles(x.dir.RawData,[],[],fullfile(x.opts.MyPath,'External'));
```

Comment:
Tricky, note that `rawdata` may be/will be read-only, when we go to the processing part.

We need to remove all these if `x.opts.bLoadData` etc in the future, to make things modular and easier to read.